### PR TITLE
982 create wizard line

### LINKED ISSUE #982: Add create wizard for `Line` element into wizard-library
As a user of OpenSCD I want to be able to add the `Line` element in the substation plugin

**Requirement**
- add create wizard for the `Line` element to the wizard library
- make sure that all potential parent element allow to trigger this create wizard: `SCL`, `Substation`

- reuse the `Voltage` element usage in the wizard as with the edit wizard for the `VoltageLevel` element

**Restrictions**
| name | required | type | pattern or restriction | default | helper text |
| --- | --- | --- | --- | --- | --- |
| name |  yes |  xs:normalizedString | min. lenght 1 | no |scl.name |
| desc | no | xs:normalizedString | no | no | scl.desc |
| type | no | xs:normalizedString | min length 1 | no | scl.type | 
| nomFreq | no | xs:decimal | min 0 | no | voltagelevel.wizard.nomFreqHelper | 
| numPhases | no | xs:unsignedByte | min 0 | no | voltagelevel.wizard.numPhaseHelper |

### PR DESCRIPTION
**Pull request by marcvanraalte**
closes #982

### CODE REVIEW

**Comment by juancho0202** [comment]
Please also resolve conflicts in this branch @marcvanraalte

File: src/editors/substation/zeroline-pane.ts [hunk 1]
```diff
@@ -155,23 +164,50 @@ export class ZerolinePane extends LitElement {
...
             <mwc-icon-button
-              id="createsubstation"
               icon="playlist_add"
-              @click=${() => this.openCreateSubstationWizard()}
-            ></mwc-icon-button>
-          </abbr>
+              @click=${() => (this.addMenu.open = true)}
+            ></mwc-icon-button
+            ><mwc-menu
+              corner="BOTTOM_RIGHT"
```
[the remark this answers is not in the document]
Comment (juancho0202) [reply src/editors/substation/zeroline-pane.ts:203]: Also add to the css the following rule in order to achieve the visual desired result of the menu:
```css
:host {
  display: block;
  position: relative;
}
```

**Comment by juancho0202** [comment]
After a discussion with @JakobVogelsang regarding the `zerloline-pane` we discovered a bug regarding the arrays that are used in the render functions. With the current queries we could be showing Substation or Line elements twice. 

Please make sure to update in the `renderSubstation()` method:
```ts
Array.from(this.doc.querySelectorAll('Substation') ?? [])
```
to 
```ts
Array.from(this.doc.querySelectorAll(':root > Substation') ?? [])
```
and in the `renderLine()` method change:
```ts
Array.from(this.doc.querySelectorAll('Line') ?? [])
```
to
```ts
Array.from(this.doc.querySelectorAll(':root > Line') ?? [])
```

File: test/integration/editors/substation/zeroline-pane.test.ts [hunk 2]
```diff
@@ -70,22 +67,4 @@ describe('zeroline-pane wizarding editing integration', () => {
       doc.querySelectorAll('ReportControl').length
     );
   });
-
-  it('add Substation element with createSubstationWizard', async () => {
```
[the remark this answers is not in the document]
Comment (marcvanraalte) [reply test/integration/editors/substation/zeroline-pane.test.ts:74]: Test is added

File: src/editors/substation/zeroline-pane.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcvanraalte) [reply src/editors/substation/zeroline-pane.ts]: done!

**Review by juancho0202 — APPROVED** [review]
LGTM!

**Pull request merged**